### PR TITLE
New Navbar

### LINKED ISSUE #539: Show full name and avatar in the navbar
### Describe the solution you'd like

Show full name in the navbar, if there is no name we will fall back to the username (login)

We can also show user avatar from GitHub

### Additional context

Screenshot

**We need to enable `avatar` and `name` property in GitHub API @ Onegraph**

If ok, I'm ready to work on this!

### LINKED ISSUE #540: Better navbar design
### Is your feature request related to a problem? Please describe.

Currently, the navbar seems little flat

### Describe the solution you'd like

We can design a navbar which is more accessible in terms of functionality and design

### Describe alternatives you've considered

- GitHub's Navbar
- Twitter's Old Navbar

If ok, I'm ready to work on this!

### PR DESCRIPTION
**Pull request by yg**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

New Navbar Design

## Related Tickets & Documents

Closes #539, Closes #540 

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by yg** [comment]
@bdougie Any feedback on this?

Screenshot

The logged out navbar doesn't display the icons, I don't know what is the reason!

File: src/components/Nav.js [hunk 1]
```diff
@@ -5,29 +5,53 @@ import {SubtleLink} from "../styles/Typography";
...
         <li>
+          <span>
+            <Octicon verticalAlign="middle" icon={getIconByName("repo")} />
+          </span>
           <SubtleLink href="https://dev.to/t/opensauced">Blog</SubtleLink>
         </li>
         <li>
+          <span>
+            <Octicon verticalAlign="middle" icon={getIconByName("mark-github")} />
+          </span>
```
Comment (bdougie) [inline src/components/Nav.js:27]: This does not align with GitHub's styleguide. https://github.com/logos

The octomark and the name GitHub are not meant to appear next to each other.

File: src/components/Nav.js [hunk 2]
```diff
@@ -5,29 +5,53 @@ import {SubtleLink} from "../styles/Typography";
...
   return (
     <FloatLeft>
+      <Link style={{verticalAlign: "middle"}} to="/">
+        <img alt="open sauced" src={logo} />
+      </Link>
       <ul>
         <li>
+          <span>
+            <Octicon verticalAlign="middle" icon={getIconByName("repo")} />
+          </span>
```
Comment (bdougie) [inline src/components/Nav.js:21]: remove the icon

File: src/components/Nav.js [hunk 3]
```diff
@@ -5,29 +5,53 @@ import {SubtleLink} from "../styles/Typography";
...
+          <span>
+            <Octicon verticalAlign="middle" icon={getIconByName("mark-github")} />
+          </span>
           <SubtleLink href="https://github.com/bdougie/open-sauced">GitHub</SubtleLink>
         </li>
         {user && (
           <li>
+            <span>
+              <Octicon verticalAlign="middle" icon={getIconByName("issue-opened")} />
+            </span>
```
Comment (bdougie) [inline src/components/Nav.js:34]: remove the icon

File: src/components/Nav.js [hunk 4]
```diff
@@ -5,29 +5,53 @@ import {SubtleLink} from "../styles/Typography";
...
+              </span>
+              <SubtleLink onClick={handleLogOut}>Logout</SubtleLink>
+            </div>
           ) : (
-            <SubtleLink onClick={handleLogIn}>Login</SubtleLink>
+            <div>
+              <span>
+                <Octicon verticalAlign="middle" icon={getIconByName("sign-out")} />
+              </span>
+              <SubtleLink onClick={handleLogIn}>Login</SubtleLink>
```
Comment (bdougie) [inline src/components/Nav.js:53]: remove the icon

**Comment by yg** [comment]
:wastebasket: Removed all icons @bdougie

**Comment by bdougie** [comment]
I marked this as ready to review by accident @yq. Is this good to merge?

**Comment by yg** [comment]
Yes @bdougie! Good to go :tada:

**Pull request merged**